cmd/clusterawsadm/cmd/gc/disable.go

```diff
@@ -5,7 +5,7 @@ Licensed under the Apache License, Version 2.0 (the "License");
 you may not use this file except in compliance with the License.
 You may obtain a copy of the License at
 
-    http://www.apache.org/licenses/LICENSE-2.0
+	http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
I see some tab changes added after rebasing GC changes, could you revert it?

Reply:
These are the new files, added with GC PRs, linter was failing. Boilerplate was not matching with the existing ones.